config/rewrites.d/developer.rackspace.com.json

```diff
@@ -320,9 +320,16 @@
           "status": 301
         },
         {
-          "description": "Redirect RPC URL to current version",
-          "from": "^/docs/private-cloud/rpc/?$",
-          "to": "/docs/private-cloud/rpc/v13",
+          "description": "Redirect unversioned RPC URL to current version",
+          "from": "^\\/docs\\/private-cloud\\/rpc\\/?$",
+          "to": "/docs/private-cloud/rpc/v14",
+          "rewrite:": false,
+          "status": 301
+        },
+        {
+          "description": "Redirect older RPC URLs to current version",
+          "from": "^\\/docs\\/private-cloud\\/rpc\\/\bv(9|1[01])\b\\/?",
```

Comment:
you've got an extra `b`, and I'd add a catchall at the end so anything linking to sections of the old guide would be redirected to the new one's front page. Change to `^\\/docs\\/private-cloud\\/rpc\\/v(9|1[01])\b\\/?(.*)$`